cluster-autoscaler/simulator/dynamicresources/snapshot/snapshot_slice_lister.go

```diff
@@ -24,6 +24,6 @@ type snapshotSliceLister struct {
 	snapshot *Snapshot
 }
 
-func (sl snapshotSliceLister) List() ([]*resourceapi.ResourceSlice, error) {
+func (sl snapshotSliceLister) ListWithDeviceTaintRules() ([]*resourceapi.ResourceSlice, error) {
```

Comment:
Could you add a `TODO(DRA): Actually handle the taint rules.` here?

Reply:
Added, sorry, forgot.